syncapi/storage/current_room_state_table.go

```diff
@@ -84,7 +84,10 @@ const selectStateEventSQL = "" +
 	"SELECT event_json FROM syncapi_current_room_state WHERE room_id = $1 AND type = $2 AND state_key = $3"
 
 const selectEventsWithEventIDsSQL = "" +
-	"SELECT added_at, event_json FROM syncapi_current_room_state WHERE event_id = ANY($1)"
+	// TODO: The session_id and transaction_id blanks are here because otherwise the rowsToStreamEvents
+	//       expects there to be exactly four columns. We need a better solution to this long term.
```

Comment:
Maybe also mention that we need to figure out whether we actually need to keep track of `session_id` and `transaction_id` in the db